src/pybind/mgr/dashboard/run-promtool-unittests.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -ex
+
+SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
+: ${CEPH_ROOT:=$SCRIPTPATH/../../../../}
+
+docker run --rm \
+    -v "$CEPH_ROOT":/ceph \
+    --name=promtool \
+    --network=host \
+    dnanexus/promtool:2.9.2 \
```

Comment:
I haven't checked, but maybe there's a way to get promtool from the Prometheus folks directly? I'd prefer that.

Reply:
@p-se unfortunately Prometheus is not widely packaged as it seems that containerized deployment is the preferred way...

In any case, I'd move this to the `install-deps.sh` stage, together with packages, pip, etc.

BTW this might be troublesome due to Dockerhub pull limits. @djgalloway @tchaikov what do you think about this? Any suggestion?

Reply:
@epuertat My bad. What I meant was this:

```
docker run --rm -it --entrypoint promtool prom/prometheus:v2.9.2
```

Reply:
All of the Jenkins builders inside the Ceph lab are configured to try and internal dockerhub mirror first so it might be okay.